### PR TITLE
Adds a new negative quirk, clownphobia

### PR DESCRIPTION
**Pull request by improvedname**
Clowns are really scary ok?

This adds a unique quirk element that the clown can impact since hes regularly around the station fear the clown with your life 

## Changelog
:cl: Improvedname
add: Adds clownphobia as an quirk
/:cl:

### CODE REVIEW

**Comment by Couls** [comment]
this is the PR that lets us roleplay IT

File: code/datums/traits/negative.dm [hunk 1]
```diff
@@ -225,6 +225,20 @@
...
+	desc = "You fear clowns with your life"
+	value = -1
+	mob_trait = TRAIT_CLOWNPHOBIA
+	gain_text = "<span class='danger'>Clowns starts to get you on your nerves.</span>"
+	lose_text = "<span claass='notice'>Clowns no longer get on your nerves.</span>"
+	medical_record_text = "Patient has developed a phobia towards clowns."
+
+/datum/quirk/clownphobia/add()
+	var/mob/living/carbon/human/H = quirk_holder
+	H.gain_trauma(/datum/brain_trauma/mild/phobia, TRUE, "clowns")
```
Comment (nemvar) [inline code/datums/traits/negative.dm]: This gets cured by brain surgery.

**Comment by Michael-Ashfield** [comment]
What happens if you play as a clown with clownphobia?

**Comment by HomocidlePancakes** [comment]
So mimes are not scary in your opinion, huh...

File: code/datums/traits/negative.dm (hunk 1 above)
Comment (actioninja) [reply code/datums/traits/negative.dm]: Getting a lobotomy to cure your fear of clowns sounds hilarious tbh

**Comment by cacogen** [comment]
strange this wasn't in

**Comment by RaveRadbury** [comment]
> strange this wasn't in

It already is in as a part of the 0 cost phobia quirk, which is 0 cost to keep people from gaming the system.

**Comment by Krysonism** [comment]
Shouldn't this be called coulrophobia?

File: code/datums/traits/negative.dm (hunk 1 above)
Comment (nemvar) [reply code/datums/traits/negative.dm]: It does, but normal brain surgery also cures it. Requires TRAUMA_RESILIENCE_LOBOTOMY

**Comment by improvedname** [comment]
F

**Pull request closed without merging**